docs/source/api/symbolic/symbolic_index.rst

```diff
@@ -1,12 +1,15 @@
 Symbolic Relativity Module
 ==========================
 
-This module contains the classes for performing symbloic 
+This module contains the classes for performing symbolic
 calculations related to General Relativity.
 
 .. toctree::
     :maxdepth: 2
 
-    metric
     christoffel
+    metric
+    ricci
     riemann
+    tensor
```

Comment:
Remove tensor from here, as the corresponding `.rst` file is not there. It is meant to be used as a base class not not meant to be exposed.

Reply:
> @OVSofia the Docs CI is failing!
> 
> Here is why: https://circleci.com/gh/einsteinpy/einsteinpy/3664?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

This is why it's failing